scripts/auto-bootstrap.sh

```diff
@@ -6,11 +6,13 @@ RED=$(tput setaf 1)
 GREEN=$(tput setaf 2)
 YELLOW=$(tput setaf 3)
 NC=$(tput sgr0) # No Color
-timestamp="/usr/local/bin/devxp-files/.timestamp"
+timestamp="$HOME/.local/devxp-files/.timestamp"
```

Comment:
I would place this files in their own place.
.local/devxp/data/.timestamp ?

Then user would be .local/devxp/data/.user ?